provisioning_templates/snippet/blacklist_kernel_modules.erb

```diff
@@ -0,0 +1,12 @@
+<%#
+kind: snippet
+name: blacklist_kernel_modules
+model: ProvisioningTemplate
+snippet: true
+%>
+
+<% if host_param('blacklist_kernel_modules') %>
+  <% host_param('blacklist_kernel_modules').split.each do |mod| %>
```

Comment:
Can you use `-%>` here and above to prevent empty lines? This will create blank line on every rendered template as well.